harness/determined/deploy/aws/constants.py

```diff
@@ -15,6 +15,8 @@ class defaults:
     REGION = "us-west-2"
     STACK_TAG_KEY = "managed-by"
     STACK_TAG_VALUE = "determined"
+    DB_SIZE = 20
+    DB_CLASS = "db.m7g.large"
```

Comment:
I would prefer if we were to rename this to `DB_INSTANCE_CLASS` so it's more clear what this is without having to check the defaults